### ISSUE TITLE
Should reactions ignore existing entities?

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/hooks-api#67]
From https://github.com/WordPoints/hooks-api/issues/66#issuecomment-167881161:

> To avoid this, what we would have to do is allow the event to fire only once, hit or miss. After that, no matter whether the event was reversed, later fires would be ignored. And if any reactions were added later, they'd have to be taken care of with the retroactive API. (Which makes me wonder: if a reaction isn't retroactive, should it specifically ignore events for existing entities?)

Basically how we could do this sans the retroactive API would be to listen for reverses and if a reverse occurs before a fire ignore all fires for that entity. That would require a special log of fires (misses too, not only hits).

**Comment by JDGrimes** [comment]
We never did this before, and while it isn't something that really has to be kept the same for legacy reasons (that is, for imported legacy reactions), it is a change that really isn't necessary in the initial version. We can revisit this later.

**Comment by JDGrimes** [comment]
Closing this in favor of #126/#119, since the events that would be affected by this would ultimately end up being migrated to that API instead, and it doesn't look like we really have a good reason for pursuing this at the moment, at least not sans the retroactive API.
